chain/network/src/config.rs

```diff
@@ -452,15 +473,7 @@ impl NetworkConfig {
             accounts_data_broadcast_rate_limit: rate::Limit { qps: 100., burst: 1000000 },
             snapshot_hosts_broadcast_rate_limit: rate::Limit { qps: 100., burst: 1000000 },
             routing_table_update_rate_limit: rate::Limit { qps: 10., burst: 1 },
-            tier1: Some(Tier1 {
-                // Interval is very large, so that it doesn't happen spontaneously in tests.
-                // It should rather be triggered manually in tests.
-                connect_interval: time::Duration::hours(1000),
-                new_connections_per_attempt: 10000,
-                advertise_proxies_interval: time::Duration::hours(1000),
-                enable_inbound: true,
-                enable_outbound: true,
-            }),
+            tier1,
```

Comment:
@geooner My bad, I was too eager to use default config, can you restore it to the one for tests as it was before?

Reply:
@wacban you mean before the cherry pick? https://github.com/near/nearcore/pull/13575/commits/f55fbd2bff57145435ce2a14f7f29846b0c1ae67

Reply:
this, and remove the tier1 variable, or set tier1 to this value
```suggestion
            tier1: Tier1 {
                // Interval is very large, so that it doesn't happen spontaneously in tests.
                // It should rather be triggered manually in tests.
                connect_interval: time::Duration::hours(1000),
                new_connections_per_attempt: 10000,
                advertise_proxies_interval: time::Duration::hours(1000),
                enable_inbound: true,
                enable_outbound: true,
            },
```

Reply:
🤦for some reason your comment are loading with some delay, I'll fix the commits now.
So, what I'll do is:
1. cherry pick https://github.com/near/nearcore/commit/cd77387594f8518685f41acb07fa8fccb600acfc (again)
2. add the values for the test that were missing
3. fix the format
Sounds good?